queries.sql

```diff
@@ -0,0 +1,68 @@
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
+
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species is null;
+
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
```

Comment:
- [ ] It would be better if after the rollback we verify if all records in the animals table still exist